changeset:   95430:36eca0b259e2
branch:      3.4
parent:      95427:657ebef5b291
user:        Serhiy Storchaka <storchaka@gmail.com>
date:        Sat Apr 04 10:05:48 2015 +0300
files:       Misc/ACKS Misc/NEWS Modules/_ctypes/_ctypes.c
description:
Issue #23338: Fixed formatting ctypes error messages on Cygwin.
Patch by Makoto Kato.


diff -r 657ebef5b291 -r 36eca0b259e2 Modules/_ctypes/_ctypes.c
--- a/Modules/_ctypes/_ctypes.c	Sat Apr 04 09:47:31 2015 +0300
+++ b/Modules/_ctypes/_ctypes.c	Sat Apr 04 10:05:48 2015 +0300
@@ -593,7 +593,7 @@
 #ifdef __CYGWIN__
 /* dlerror() isn't very helpful on cygwin */
         PyErr_Format(PyExc_ValueError,
-                     "symbol '%s' not found (%s) ",
+                     "symbol '%s' not found",
                      name);
 #else
         PyErr_SetString(PyExc_ValueError, ctypes_dlerror());
@@ -3279,7 +3279,7 @@
 #ifdef __CYGWIN__
 /* dlerror() isn't very helpful on cygwin */
         PyErr_Format(PyExc_AttributeError,
-                     "function '%s' not found (%s) ",
+                     "function '%s' not found",
                      name);
 #else
         PyErr_SetString(PyExc_AttributeError, ctypes_dlerror());

